README.md

```diff
@@ -111,6 +111,89 @@ func quit(g *gocui.Gui, v *gocui.View) error {
 }
 ```
 
+### Example with Simulated Test Screen
```

Comment:
I find this title a bit too vague as you have to read the complete title to understand it.  
Personally i would like to know by the first few words "oh this is about testing a gocui program".

Can you maybe change this to be something short and clear like: `Testing example`  
Or change this to more be faster understandable like: `Test gocui program using a simulated screen`?

Also can you change this to not be a parent of the example? `###` > `##`

Reply:
Yeah sounds good, I'll update this now.

Reply:
It's now `## Testing example`